README.md

```diff
@@ -12,4 +12,4 @@ and use it 3 times inside the `App` instead of static markup.
 ## Instructions
 - Implement a solution following the [React task guideline](https://github.com/mate-academy/react_task-guideline#react-tasks-guideline).
 - Open one more terminal and run tests with `npm test` to ensure your solution is correct.
-- Replace `<your_account>` with your Github username in the [DEMO LINK](https://<your_account>.github.io/react_person/) and add it to the PR description.
+- Replace `<your_account>` with your Github username in the [DEMO LINK](https://Stako21.github.io/react_person/) and add it to the PR description.
```

Comment:
Add this link to the PR description
<img width="942" alt="image" src="https://github.com/mate-academy/react_person/assets/64750958/25f91f91-b908-4e64-ac47-7684458334dd">

Reply:
still not fixed
<img width="603" alt="image" src="https://github.com/mate-academy/react_person/assets/70220667/4d64e381-0b06-4792-95fb-23015a5ba55d">